libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/InstructionView.java

```diff
@@ -334,6 +381,33 @@ public void hideInstructionList() {
     onInstructionListVisibilityChanged(false);
   }
 
+  /////////////////////////////////////////
+  private void showGuidanceView(double distanceTraveled, double currentStepTotalDistance) {
+    int thresholdToShowGuidanceView = 200;
```

Comment:
Add unit into the variable name? 200 what? Meters?

Reply:
That's a good question. I think it should be `meters` irrespective of the distance unit one is following. It should go into the documentation.